tls-client/README.md

```diff
@@ -24,42 +24,42 @@ The output in the terminal window should be similar to this:
 
 ```
 Using Ethernet LWIP
-Client IP Address is 10.2.203.43
-Connecting with developer.mbed.org
+Client IP Address is 172.16.8.12
+Connecting with os.mbed.com
 Starting the TLS handshake...
-TLS connection to developer.mbed.org established
+TLS connection to os.mbed.com established
 Server certificate:
     cert. version     : 3
-    serial number     : 11:21:B8:47:9B:21:6C:B1:C6:AF:BC:5D:0C:19:52:DC:D7:C3
+    serial number     : 65:7B:6D:8D:15:A5:B6:86:87:6B:5E:BC
     issuer name       : C=BE, O=GlobalSign nv-sa, CN=GlobalSign Organization Validation CA - SHA256 - G2
     subject name      : C=GB, ST=Cambridgeshire, L=Cambridge, O=ARM Ltd, CN=*.mbed.com
-    issued  on        : 2016-03-03 12:26:08
-    expires on        : 2017-04-05 10:31:02
+    issued  on        : 2017-04-03 13:54:02
+    expires on        : 2018-05-06 10:31:02
     signed using      : RSA with SHA-256
     RSA key size      : 2048 bits
     basic constraints : CA=false
     subject alt name  : *.mbed.com, mbed.org, *.mbed.org, mbed.com
+    subject alt name  : *.mbed.com, mbed.org, *.mbed.org, mbed.com
```

Comment:
This line got duplicated (I guess the intend was to remove the whitespace in the end?)